reporting-app/app/models/certification.rb

```diff
@@ -68,4 +68,8 @@ def origin
   def member_name
     self&.member_data&.name
   end
+
+  def region
+    self&.certification_requirements&.region
```

Comment:
I believe `self` isn't necessary on instance methods. 

```suggestion
  certification_requirements&.region
```

Reply:
Yeah, I was just following the convention already set in the file. It seems like other instance methods are using `self`.